js-wixcode-sdk/$w/VideoBox.service.json

```diff
@@ -0,0 +1,608 @@
+        "ret":
+          { "type": "$w.VideoBox",
+            "doc": "The video box that triggered the event." },
+        "locations":
+          [ { "lineno": 418,
+              "filename": "VideoBox.es6" } ],
+        "docs":
+          { "summary": "Adds an event handler that runs when playback is started or restarted.",
+            "description":
+              [ "A video in a video box can be played by a user clicking the video box's",
+                " play button or by calling its [`play()`](#play) function." ],
+            "links":
+              [ "[play( )](#play)" ],
+            "examples":
+              [ { "title": "Get the ID of the video box that is playing",
+                  "body":
+                    [ "$w(\"#myVideoBox\").onPlay( (event) => {",
+                      "  let targetId = event.target.id; // \"myVideoBox\"",
+                      "});" ],
+                  "extra":
+                    {  } } ],
+            "extra":
+              {  } },
+        "extra":
+          { "eventType": "onPlay" } },
+      { "name": "onProgress",
+        "labels": [],
+        "nameParams": [],
+        "params":
+          [ { "name": "handler",
+              "type": "$w.EventHandler",
+              "doc": "The name of the function or\n the function expression to run when the video progresses." } ],
+        "ret":
+          { "type": "$w.VideoBox" },
+        "locations":
+          [ { "lineno": 552,
+              "filename": "VideoBox.es6" } ],
+        "docs":
+          { "summary": "Adds an event handler that runs when playback progresses.",
+            "description":
+              [ "A video continually progresses as the video plays." ],
+            "links": [],
+            "examples":
+              [ { "title": "Get the ID of the video box that is progressing",
+                  "body":
+                    [ "$w(\"#myVideoBox\").onProgress( (event) => {",
+                      "  let targetId = event.target.id; // \"myVideoBox\"",
+                      "});" ],
+                  "extra":
+                    {  } } ],
+            "extra":
+              {  } },
+        "extra":
+          { "eventType": "onProgress" } },
+      { "name": "pause",
+        "labels": [],
+        "nameParams": [],
+        "params": [],
+        "ret":
+          { "type":
+              { "name": "Promise",
+                "typeParams":
+                  [ "void" ] },
+            "doc": "Fulfilled - When playback has been paused." },
+        "locations":
+          [ { "lineno": 260,
+              "filename": "VideoBox.es6" } ],
+        "docs":
+          { "summary": "Pauses playback.",
+            "description":
+              [ "The `pause()` function pauses the current video and fires a `pause` event.",
+                "",
+                " The `pause` event handlers set on the video box by the [onPause( )](#onPause)",
+                " function or in the Editor are called." ],
```

Comment:
What does that mean the 'in the Editor'? If you don't know either just remove it.